cadc-vos-server/src/main/java/ca/nrc/cadc/vos/server/NodeID.java

```diff
@@ -82,20 +82,35 @@
     Long id;
     Subject owner;
     Object ownerObject;
+    String storageID;
 
     public NodeID() { }
-    
+
+    /**
+     * NodeID constructor.
+     *
+     * @param id
+     * @param owner
+     * @param ownerObject
+     */
+    public NodeID(Long id, Subject owner, Object ownerObject) {
+        this(id, owner, ownerObject, null);
+    }
+
     /**
      * NodeID constructor.
      * 
      * @param id
      * @param owner
+     * @param ownerObject
+     * @param storageID
      */
-    public NodeID(Long id, Subject owner, Object ownerObject)
+    public NodeID(Long id, Subject owner, Object ownerObject, String storageID)
```

Comment:
Is this constructor (and the one above with one less param) actually used anywhere?

Reply:
The 3-arg constructor is used in a unit test. The 4-arg constructor isn't used. I updated assuming both were used in code somewhere, but then never found that somewhere.

Reply:
Removed the 4-arg constructor.